src/main/java/roomescape/reservation/domain/Reservation.java

```diff
@@ -18,49 +21,42 @@ public class Reservation {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
     private Member member;
 
-    @ManyToOne
-    private ReservationTime reservationTime;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
+    private TimeSlot timeSlot;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
     private Theme theme;
 
     @Column(nullable = false)
     private LocalDate date;
 
-    private Reservation(final Long id, final Member member, final LocalDate date, final ReservationTime reservationTime,
+    private Reservation(final Long id, final Member member, final LocalDate date, final TimeSlot timeSlot,
                         final Theme theme) {
-        validateNull(member, date, reservationTime, theme);
+        validateNull(member, date, timeSlot, theme);
         this.id = id;
         this.member = member;
         this.date = date;
-        this.reservationTime = reservationTime;
+        this.timeSlot = timeSlot;
         this.theme = theme;
     }
 
-    public Reservation() {
+    protected Reservation() {
     }
 
-    public static Reservation of(final Long id, final Member member, final LocalDate date,
-                                 final ReservationTime reservationTime, final Theme theme) {
-        return new Reservation(id, member, date, reservationTime, theme);
-    }
-
-    public static Reservation createWithoutId(final Member member, final LocalDate date,
-                                              final ReservationTime reservationTime, final Theme theme) {
-        return new Reservation(null, member, date, reservationTime, theme);
-    }
-
-    private static void validateNull(Member member, LocalDate date, ReservationTime reservationTime, Theme theme) {
+    private static void validateNull(Member member, LocalDate date, TimeSlot timeSlot, Theme theme) {
```

Comment:
static 일 필요가 없겠네요!
`Waiting`도 같이 반영해주세요~

```suggestion
    private void validateNull(Member member, LocalDate date, TimeSlot timeSlot, Theme theme) {
```

Reply:
넵 반영했습니다!